docs/platforms/react-native/manual-setup/expo/expo-sagp.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Sentry Android Gradle Plugin (AGP)
+description: "Learn how to configure the Sentry Android Gradle Plugin to automatically upload source maps and debug symbols for your Expo project."
+---
+
+<Alert>
+
+The **Sentry AGP** is still an experimental option of the Sentry React Native Expo plugin. If you encounter any issues please let us know via a GitHub ticket.
+
+</Alert>
+
+To enable the plugin modify the Expo application configuration and add the `enableAndroidGradlePlugin` option in the `@sentry/react-native/expo` plugin section.
+
+```javascript {tabTitle:app.json/app.config.json}
+{
+  "expo": {
+...
+    "plugins": [
+      [
+        "@sentry/react-native/expo",
+        {
+          "url": "https://sentry.io/",
+          "project": "project-slug",
+          "organization": "organization-slug",
+          "experimental_android": {
+            "enableAndroidGradlePlugin": true,
+          }
+        }
+      ],
+...
+  }
+}
+```
+
+## Configuration options
+
+The supported [Sentry Android Gradle Plugin options](/platforms/android/configuration/gradle/) are the following:
+- `autoUploadProguardMapping`
+- `includeProguardMapping`
+- `dexguardEnabled`
+- `uploadNativeSymbols`
+- `autoUploadNativeSymbols`
+- `includeNativeSources`
+- `includeSourceContext`
+
```

Comment:
I would replace the list of the options with a section under the code snippet of the options with a description of the functionality.

For example how we do it in https://github.com/getsentry/sentry-docs/blob/5873e415e9cedf3d911a6f79305554fa8981804f/docs/platforms/react-native/tracing/instrumentation/react-native-navigation.mdx#L27-L54

The description already exists in https://sentry-docs-3s3vjrwvt.sentry.dev/platforms/android/configuration/gradle/ so we can just copy it.

Reply:
Good idea @krystofwoldrich 👍 
Updated with 193fa06ece43bac4f909b4d804df1b17e56d6908